### DISCUSSION TITLE
How to remove overly color from drawer component?

### DISCUSSION THREAD

**Discussion by absoftlabs** [saadeghi/daisyui#3069] [Q&A] [answered]
When I remove drawer-overlay class, it removes the overlay color but doesn't work the close button. can anyone help me remove the overlay color from the drawer component of the daisy UI?

**Comment by saadeghi** [comment, marked as answer]
Don't remove it. It's not just the color, it's also size and position of that part.  

To change the color to transparent, add `bg-transparent` to `drawer-overlay`

Let me know if you have a question.
